### PR TITLE
Bug 1105650 - [Stingray] Apply visual of button on app-deck list

### PR DESCRIPTION
**Pull request by huchengtw-moz**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[huchengtw-moz John Hu (huchengtw-moz)](https://github.com/huchengtw-moz) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=ee645e820f26)

**Comment by try-server-hook** [comment]
[huchengtw-moz John Hu (huchengtw-moz)](https://github.com/huchengtw-moz) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=2fff3e13ba12)

**Comment by try-server-hook** [comment]
[huchengtw-moz John Hu (huchengtw-moz)](https://github.com/huchengtw-moz) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=9f04778656df)

File: tv_apps/app-deck/js/promotion_list.js [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+      // If context item is undefined or null, it's first context item, we just
+      // set it without animation.
+      this._contextItem = item;
+      this._fillMainBanner(this.container, item);
+      return;
+    }
+
+    if (this._contextItem === item) {
+      this._cancelSliding();
+      console.log('the same item found: ' + idx);
```
Comment (dwi2) [inline tv_apps/app-deck/js/promotion_list.js]: nit: I think we could remove console log of debugging purpose. what do you think?

File: tv_apps/app-deck/js/promotion_list.js [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+      return;
+    }
+
+    if (this.animator.classList.contains('sliding')) {
+      this._pendingAnimationItem = item;
+      return;
+    }
+    console.log('slide in' + idx);
```
Comment (dwi2) [inline tv_apps/app-deck/js/promotion_list.js]: nit: same as above

**Comment by try-server-hook** [comment]
[huchengtw-moz John Hu (huchengtw-moz)](https://github.com/huchengtw-moz) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=b0fa843955ad)

**Pull request merged**